clang/docs/ReleaseNotes.rst

```diff
@@ -555,6 +555,8 @@ Improvements to Clang's diagnostics
       getS(); // Now diagnoses "Reason 2", previously diagnoses "Reason 1"
     }
 
+- Clang now omits warnings for constructors marked with the ``[[noreturn]]`` attribute (#GH63009).
```

Comment:
```suggestion
- Clang now correctly recognises code after a call to a ``[[noreturn]]`` constructor
  as unreachable (#GH63009).
```
I think that’s a bit clearer; the original release note makes it sound like we stop warning about `noreturn` altogether.

Reply:
@Sirraide thanks. I've updated.